### PR TITLE
CLOUDP-112245: [Milestone] Atlascli/Mongocli Doc changes

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-112245

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

**Description:**
Changes:

- This PR moves the docs file from `docs/commands` to `docs/mongocli/commands` -> **This is a breaking change for the doc tool**
- We generated the atlascli doc in `docs/atlascli/commands`

### CODE REVIEW

File: internal/docs/atlascli/main.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```
Comment (andreaangiolillo) [inline internal/docs/atlascli/main.go]: This is the second change

File: internal/docs/mongocli/main.go [hunk 2]
```diff
@@ -31,13 +31,13 @@ func setDisableAutoGenTag(cmd *cobra.Command) {
 }
 
 func main() {
-	if err := os.RemoveAll("./docs/command"); err != nil {
+	if err := os.RemoveAll("./docs/mongocli/command"); err != nil {
```
Comment (andreaangiolillo) [inline internal/docs/mongocli/main.go]: This is the last change

**Review by gssbzn — COMMENTED** [review]
is docs prepare for this change? what happens if we need to release tomorrow?

**Comment by andreaangiolillo** [comment]
they are a reviewer of this PR so we will merge this only when they are ready

**Review by sarahsimpers — APPROVED** [review]
LGTM from docs side, thanks!

**Pull request merged**